.github/container/test-maxtext.sh

```diff
@@ -248,6 +248,7 @@ if [ -z "$DECODER_BLOCK" ]; then
     RUN_SETTINGS="MaxText/train.py MaxText/configs/base.yml run_name=${RUN_NAME} model_name=${MODEL}\
         steps=$STEPS per_device_batch_size=${BATCH_PER_GPU} remat_policy=${REMAT_POLICY} enable_checkpointing=false\
         base_output_directory=$OUTPUT dataset_path=local dataset_type=synthetic hardware=$HARDWARE\
+        enable_goodput_recording=false monitor_goodput=false\
```

Comment:
nit: I think it would be better to expand RUN_SETTINGS so that it's one key=value per line for readability and maintenance

Reply:
@terrykong Done!